examples/only-logs-otel/rendered_manifests/configmap-agent.yaml

```diff
@@ -155,7 +155,7 @@ data:
         - id: parser-crio
           regex: ^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) ?(?P<log>.*)$
           timestamp:
-            layout: "2006-01-02T15:04:05.999999999-07:00"
+            layout: 2006-01-02T15:04:05.999999999Z07:00
```

Comment:
It's unfortunate format constants like `RFC3339Nano` can't be used.

Reply:
+1